gitea/statefulset.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: gitea
+  namespace: gitea
+  labels:
+    app: gitea
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: gitea
+  template:
+    metadata:
+      labels:
+        app: gitea
+    spec:
+      securityContext:
+        fsGroup: 1000
+      ##  runAsUser: 1000
+      ##  runAsGroup: 1000
+      containers:
+      - name: gitea
+        image: gitea/gitea:latest
+        env:
+          - name: SSH_LISTEN_PORT
+            value: "22"
+          - name: SSH_PORT
+            value: "22"
+          - name: GITEA_APP_INI
+            value: /data/gitea/conf/app.ini
```

Comment:
Should be nice to include the `app.ini` content into a ConfigMap resource?

Reply:
Yeah, that would be another future refinement, perhaps. @yuwenma did some work in the past to make `ini` files editable in the kpt function pipeline like KRM resources. But I don't know if we ever merged it.

Reply:
Unfortunately no. I captured the solutions, related materials and POC code, future plans here https://docs.google.com/document/d/1DJH2Age9vGplwV8udsCdCblBRurDvSg2LvMzk_jGqmw